src/Oro/Bundle/ActionBundle/Resources/config/cache.yml

```diff
@@ -32,3 +32,11 @@ services:
             - [addProvider, ['@oro_action.configuration.provider.action_groups']]
         tags:
             - { name: kernel.cache_warmer }
+
+    oro_action.config.dumper:
```

Comment:
These dumpers are used only in debug mode.
I think it will be better to register them in DIC only if debug mode is enabled.
Here is an example: https://github.com/laboro/dev/blob/master/package/platform/src/Oro/Bundle/ApiBundle/DependencyInjection/OroApiExtension.php#L74-L75